### ISSUE TITLE
Investigate extra snapshot/timestamp GHA run

### ISSUE THREAD

**Issue by Hayden-IO** [sigstore/root-signing#814]
#812 occurred on Friday, #813 occurred on Sunday. It should be occurring every week at most, so why did it retrigger?

**Description**

**Version**

**Comment by asraa** [comment]
I checked both of their event types by accessing their workflow runs with the GH API. They're both running on schedule, so I think this is a schedule-ing issue with cron. The repeats happen near the end of the month.

I don't really think this is super critical. Is the issue storage?

**Comment by asraa** [comment]
You might be able to play with the cron expression to ensure, for e.g. it only happens on Mondays

**Comment by Hayden-IO** [comment]
Not critical, mostly just wondering why, given we've seen this before.

Yea, there's a separate issue I filed in public good instance to run the job during working hours.

**Comment by asraa** [comment]
It must be something with the end of the month - i think the cron is supposed to be every 7th day of the month, and so it might end up repeating near the start of the next month by triggering on the first day.

**Comment by asraa** [comment]
If GH supports expressions like this, maybe changing it to 9 on mondays works? 0 9 * * MON

Depends on who's working hours, I think that might be UTC time.

**Comment by kommendorkapten** [comment]
This is the same issue we had when we ran into when we did the v6 root signing. The cron job is configured to run on the 7th day of the month. So on Friday it was the 28th then the first day of the month would also be considered the a 7th (0/7 mod 7 == 0).

This was also discussed here where I proposed to change the cron job definition: https://github.com/sigstore/root-signing/issues/693#issue-1605062919
